sycl/test/graph/graph-explicit-repeated-exec.cpp

```diff
@@ -37,12 +35,14 @@ int main() {
   }
 
   q.submit([&](sycl::handler &h) { h.ext_oneapi_graph(executable_graph); });
+  q.wait();
```

Comment:
what would make this test better is actually if we did had an out of order queue and did:
```cpp
auto e1 = q.submit([&](sycl::handler &h) { h.ext_oneapi_graph(executable_graph); });
q.submit([&](sycl::handler &h) { h.depends_on(e1); h.ext_oneapi_graph(executable_graph); });
q.wait()
```

This this doesn't pass or causes problems then is fine as-is though

Reply:
Hmm, that would be possible but then we wouldn't be able to verify the results in between graph executions. Perhaps some combination of both would work though.

Reply:
Good point, this is really an idea for a separate test. Resolving as out-of-scope.